prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -70,6 +70,8 @@ struct rxr_req_inf REQ_INF_LIST[] = {
 	[RXR_LONG_TAGRTM_PKT] = {4, sizeof(struct rxr_long_tagrtm_hdr), 0},
 	[RXR_READ_MSGRTM_PKT] = {4, sizeof(struct rxr_read_msgrtm_hdr), RXR_REQ_FEATURE_RDMA_READ},
 	[RXR_READ_TAGRTM_PKT] = {4, sizeof(struct rxr_read_tagrtm_hdr), RXR_REQ_FEATURE_RDMA_READ},
+	[RXR_DC_EAGER_MSGRTM_PKT] = {4, sizeof(struct rxr_dc_eager_msgrtm_hdr), 0},
```

Comment:
```RXR_DC_EAGER_MSGRTM_PKT``` is an extra feature, and need to set the RXR_REQ_FEATURE_DELIVERY_COMPLETE feature flag

Reply:
Feature flag is set now. resolving.